app/src/kv_types.h

```diff
@@ -156,33 +144,44 @@ namespace scitt
     Policy policy = {};
     Authentication authentication = {};
 
-    // The long-term stable identifier of this service, as a DID.
+    // The long-term stable issuer string of this service.
     // If set, it will be used to populate the issuer field of receipts
-    std::optional<std::string> service_identifier;
+    std::optional<std::string> service_issuer;
```

Comment:
Same remark as above, `cose_signatures.issuer` is used for that purpose, not this value.

Reply:
add a section in the doc about `cose_signatures` and removed  `service_issuer` from python tests and from docs